pkg/reconciler/v1alpha1/route/README.md

```diff
@@ -32,7 +32,7 @@ the following objects:
 - A VirtualService to realize the routing from the Gateway
   `knative-shared-gateway` to the traffic target referenced in the Route.
```

Comment:
This has changed to `knative-ingress-gateway`, can you please update?  Thanks.

Reply:
done